src/stservo/gui/servo_gui.py

```
@@ -1544,6 +1701,280 @@ def clear_sync_results(self):
+    
+    def unlock_eprom(self):
+        """Unlock EPROM for writing"""
+        if not self.connected:
+            messagebox.showerror("Error", "Not connected to servo")
+            return
+        
+        try:
+            self.packet_handler.unLockEprom(self.current_servo_id)
+            self.eprom_status_label.config(text="EPROM Status: Unlocked", foreground="red")
+            self.log_message(f"EPROM unlocked for servo ID {self.current_servo_id}")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)